src/python/gem5/components/boards/riscv_board.py

```diff
@@ -256,7 +256,7 @@ def generate_device_tree(self, outdir: str) -> None:
             root.append(node)
 
         node = FdtNode(f"chosen")
-        bootargs = " ".join(self.get_default_kernel_args())
+        bootargs = self.workload.boot_args
```

Comment:
After #630 is merged, I think this should be
```suggestion
        bootargs = self.workload.command_line
```

We should double check with the `ArmBoard` to see what it uses.